jsFiles/utility.js

```diff
@@ -63,17 +63,39 @@ function alertError(error) {
  *
  * @param {number} timeStamp - unix, contains information about the time.
  */
-// TODO: fix for when displaying the time of a searched city -- timezones offset
-function timeConversion(timeStamp) {
-  // The multiplication *1000 is because the timeStamp is in second
-  // and the Date expects miliseconds
-  const timeObj = new Date(timeStamp * 1000);
-  return timeObj.toLocaleTimeString("en-US", {
+function timeConversion(timeStamp, offsetTime) {
+  // The multiplication *1000 is because the timeStamp and offsetTime are in second
+  // and the Date object expects miliseconds.
+  const timeObjWithOffset = new Date(timeStamp * 1000 + offsetTime * 1000);
+  return formatTime(timeObjWithOffset, "UTC");
+}
+
+/**
+ * this function format time --> hh:mm AM/PM
+ *
+ * @param {object} dateObj - contains date and time information
+ * @param {string} timeZone - "UTC" or undefined as default for local time
+ */
+//
+function formatTime(dateObj, timeZone = undefined) {
+  // en-US: uses 12-hour time with AM/PM
+  // timeZone: timeZone  == timeZone --> just because I used the same name
+  return dateObj.toLocaleTimeString("en-US", {
```

Comment:
you can call it just date. It should be obvious that this should be an object :)

Reply:
Thanks for your time reviewing :)